File: desafio-03/mayrloncarlos/java/desafio03.java
package teste;

public class desafio03 {


	public static void main(String[] args) {
		
		int resposta = 0;
		
		for(int i=1;i<=100000;i++) {
			int n = i;
			while(n > 0) {
				resposta += n % 10;
				resposta *= 10;
				n /= 10;
			}
			int numeroFinal = resposta/10;
			
			n = i;
			resposta = 0;
			
			if(n == numeroFinal) {
				System.out.println(n + " é palíndromo");
			} else {
				System.out.println(n + " não é palíndromo");
			}	
		}
	}
}

File: desafio-03/JacksonOsvaldo/python/desafio03.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-


"""
Resposta de Jackson Osvaldo da Silva Braga
GitHub: https://github.com/JacksonOsvaldo
E-mail: jacksonosvaldo@live.com
"""


def negative(inicio, fim):
    """Copy of the list in reverse order"""
    for numero in range(inicio, fim+1):
        num = str(numero)
        if num == num[::-1]:
            print('Palíndromo: {}'.format(num))

negative(1, 100000)
